packages/zendesk-adapter/src/filters/custom_objects/custom_object_fields.ts

```diff
@@ -372,7 +373,7 @@ const customObjectFieldsFilter: FilterCreator = ({ config, client }) => {
     },
     // Knowing if a value is a user depends on the custom_object_field attached to its condition's field
     // For that reason we need to specifically handle it here, using 'is_user_value' field that we added in onFetch
-    preDeploy: async (changes: Change<InstanceElement>[]) => {
```

Comment:
I think its a bad merge, fixing

Reply:
it wasn't fixed

Reply:
i got confused when i answered, i changed it to "onDeploy: async changes => {" because the previous one was incorrect
"onDeploy: async changes => {" is the correct one